src/api/index.spec.tsx

```diff
@@ -189,6 +189,20 @@ describe(__filename, () => {
         expect.any(Object),
       );
     });
+
+    it('can include credentials', async () => {
+      await callApiWithDefaultApiState({
+        endpoint: '/url',
+        includeCredentials: true,
```

Comment:
It would be nice to get an additional test to make sure `credentials` is undefined without this.